scripts/wsrep_sst_mariabackup.sh

```diff
@@ -76,7 +77,7 @@ sdecomp=""
 # 5.6.21 PXC and later can't donate to an older joiner
 sst_ver=1
 
-if which pv &>/dev/null && pv --help | grep -q FORMAT;then 
+if which pv &>/dev/null && pv --help | grep -q FORMAT;then
```

Comment:
Patch contains lot of these white space changes are these really required and why ?

Reply:
Automatic removal of EOL whitespace is the default behavior of the editor being used (Atom).   This is a default behavior as having EOL whitespace is considered bad practice.

Reply:
Is this pull request going to get merged?  This fixes a significant issue for anyone using encryption for sst.